src/quilla/__init__.py

```diff
@@ -77,10 +76,29 @@ def make_parser() -> argparse.ArgumentParser:  # pragma: no cover
         '-u',
         '--update-baseline',
         dest='update_baseline',
-        action='store_true',
-        help='Used to update the baseline images for VisualParity. '
+        action='append',
+        metavar='BASELINE_ID',
+        help='Used to update the specific baseline image associated with '
+        'the provided baseline ID for VisualParity. '
         'Different plugins define different behaviour, so this is not '
-        'necessarily a lossless operation.'
+        'necessarily a lossless operation.',
+        default=[],
+    )
+    config_group.add_argument(
+        '-U',
+        '--update-all-baselines',
+        dest='update_all_baselines',
+        action='store_true',
+        help='Used to update all baseline images in a Quilla test file. '
+        'this is equivalent to using --update-baseline with every baseline ID '
+        'that is used in the Quilla test.'
+    )
+    config_group.add_argument(
+        '--create-baseline-if-none',
```

Comment:
How about "--create-baseline-if-absent"?

Reply:
Sure, that's a good name also